### ISSUE TITLE
Registration Moderators

### ISSUE THREAD

**Issue by rakshi00** [ElsaHilftDigital/main#9]

**Comment by F1rst-Unicorn** [comment]
We should not allow arbitrary people to register as moderator, so some unlocking has to happen after signup

**Comment by rakshi00** [comment]
Agreed. I'd say, one existing moderator has to approve. We would need similar front-end to helper view, I guess.
